riscv-sbi.md

```diff
@@ -55,20 +60,24 @@ environment. When executed in S-mode, it generates an environment-call-from-S-mo
 exception and performs no other operation. The required arguments are passed
 through registers *a0-a2* and the SBI call type is passed via register *a7*. Once
 the machine mode receives the trap, it identifies the type of SBI call from *a7*
-register and perform required operation. Individual SBI call signature and its
-purpose is described next.
+register and perform required operation. Any unsupported SBI call shoud return
+error i.e. -ENOSYS to indicate the supervisor mode that it is not supported.
```

Comment:
As (I think) I've said before, I'd prefer if the value of `ENOSYS` was explicitly defined, rather than relying on implicit external definitions.

Reply:
ENOSYS is a standard linux error code (38). Do you want me to just add the value as well ?

Reply:
I agree with @neuschaefer -- this is not constant across operating systems, and the SBI should support more than Linux, so we should be explicit.  (Hurd defines it to be 78, and some Newlib ports use 88.)

I suggest writing something like "Any unsupported SBI call should return -38 (i.e., -ENOSYS)."

Reply:
Sounds good to me. Will update that.